.ci.yaml

```diff
@@ -8,46 +8,113 @@
 enabled_branches:
   - master
 
+platform_properties:
+  linux:
+    properties:
+      caches: builder_linux_engine:builder
+      os: Linux
+  mac:
+    properties:
+      caches: flutter_cocoapods:cocoapods, osx_sdk:old_osx_sdk, builder_mac_engine:builder
```

Comment:
@christopherfujino does this format seem ok to you? In the flutter/infra starlark code, I special case "caches" and split it by `(", ")` and `(":")` to get the `name`, `path` for `swarming.cache`.

Reply:
- `flutter_cocoapods:cocoapods, osx_sdk:old_osx_sdk, builder_mac_engine:builder` is a single string, correct? if so, can you make that explicit with double quotes?

I was going to suggest making this stringified list parseable JSON, like:

```
caches: "[\"flutter_cocoapods:cocoapods\", \"osx_sdk:old_osx_sdk\"]"
```

But then I realize that there is no JSON.parse library in lucicfg, so we'd have to hand parse it anyway.

Reply:
Oh, maybe it is implemented now? https://chromium.googlesource.com/infra/luci/luci-go/+/refs/heads/main/lucicfg/starlark/stdlib/json_doc.star#80

I tried using this locally and it works...

```
config_str = """
{
  "platform_properties": {
    "linux": {
      "properties": {
        "caches": "builder_linux_engine:builder",
        "os": "Linux"
      }
    },
    "mac": {
      "properties": {
        "caches": "flutter_cocoapods:cocoapods, osx_sdk:old_osx_sdk, builder_mac_engine:builder"
      }
    }
  }
}
"""
config_dic = json.decode(config_str)
caches = config_dic["platform_properties"]["mac"]["properties"]["caches"]
print(caches.split(", ")[1].split(":")) # ["osx_sdk", "old_osx_sdk"]
```